api/osrd_infra/migrations/0001_initial.py

```diff
@@ -179,7 +179,7 @@ class Migration(migrations.Migration):
                         max_length=8,
                     ),
                 ),
-                ("speed_limit_composition", models.CharField(max_length=128, null=True)),
```

Comment:
You can't change an old migration. You must create a new one that actually changes the column name.